Software/src/LedDeviceAlienFx.hpp

```diff
@@ -38,20 +38,20 @@ class LedDeviceAlienFx : public AbstractLedDevice
 public:
 	LedDeviceAlienFx(QObject *parent = 0);
 	virtual ~LedDeviceAlienFx();
+	QString name() const { return QStringLiteral("lightfx"); }
+	int maxLedsCount();
+	int defaultLedsCount() { return 1; }
```

Comment:
just curious, why is the maxLedsCounts in the code file but the defaultLedsCount in the header?

Reply:
`(QString1, QString2...)` (up to 9 i think)
`arg(type1).arg(type2)...` which is probably equal to `(QString(type1), QString(type2)...)` allocation wise